### DISCUSSION TITLE
Can't till soil with hoe

### DISCUSSION THREAD

**Discussion by Inksaver** [cc-tweaked/CC-Tweaked#1556] [Q&A] [answered]
Version 1.20.1 forge: I cannot get the hoe to till soil. It used to work by placing the turtle 1 block above the dirt and using digDown(). I have tried the new digDown("left") and digDown("right") but ony reports 'nothing to dig here'. I have tried it in direct contact with the dirt: 'this tool cannot be used' and digging forward also does not work. Using placeDown() with a hoe in the inventory works, but damages the tool.

I am trying to update my toolkit for 1.20.1   https://github.com/Inksaver/Minecraft-Toolkit/tree/main but this is holding me back.

Any ideas?

**Comment by SquidDev** [comment, marked as answer]
Also reported as a bug at #1557. This was fixed in 1.107.0.
